### ISSUE TITLE
:GoAddTags json,omitempty doesn't work

### ISSUE THREAD

**Issue by Carpetsmoker** [fatih/vim-go#1777]
`:GoAddTags` is documented as:

    If [option] is passed it'll either add a new tag with an option or will
    modify existing tags. An example of adding omitempty to all json
    fields would be:

      :GoAddTags json,omitempty

But that doesn't work; `:GoAddTags json` works fine, but `:GoAddTags json,omitempty` doesn't seem to do anything.

Not a recent breakage – I noticed this over a month ago (...just forgot to create issue).

**Comment by fatih** [comment]
This is by purpose like this (another discussion from last year: https://github.com/fatih/vim-go/issues/985#issuecomment-279704742) . To add an option, the tag needs to exist. If not it won't do anything. So to add the option you have the call these two:

```
:GoAddTags json
:GoAddTags json,omitempty
```

There is a reason I did this, but I still don't remember it unfortunately. I'm ok changing it back the behavior and seeing how it feels. The reason I've added it might come up in this case.
